From e6533dbd6dde18740d5417d0d3c99315befa4f72 Mon Sep 17 00:00:00 2001
From: Jason Wessel <jason.wessel@windriver.com>
Date: Thu, 21 Nov 2019 20:27:03 -0800
Subject: [PATCH] rpi_4 defconfigs: Add CONFIG_CMD_BOOT menu

This allows the capability for easily selecting a rollback option when
using ostree and the boot.scr feature.

Upstream-Status: Inappropriate [wr-ostree specific]

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>
---
 configs/rpi_4_32b_defconfig | 2 ++
 configs/rpi_4_defconfig     | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/configs/rpi_4_32b_defconfig b/configs/rpi_4_32b_defconfig
index a31a617a5fb..d7e8208aa8e 100644
--- a/configs/rpi_4_32b_defconfig
+++ b/configs/rpi_4_32b_defconfig
@@ -31,3 +31,5 @@ CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_CONSOLE_SCROLL_LINES=10
 CONFIG_PHYS_TO_BUS=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_MENU=y
+CONFIG_CMD_BOOTMENU=y
diff --git a/configs/rpi_4_defconfig b/configs/rpi_4_defconfig
index da8c960a2a1..2c4ea708e47 100644
--- a/configs/rpi_4_defconfig
+++ b/configs/rpi_4_defconfig
@@ -31,3 +31,5 @@ CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_CONSOLE_SCROLL_LINES=10
 CONFIG_PHYS_TO_BUS=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_MENU=y
+CONFIG_CMD_BOOTMENU=y
-- 
2.23.0

